storage/rocksdb/rdb_i_s.cc

```diff
@@ -22,6 +22,7 @@
 #include <vector>
 
 /* MySQL header files */
+#define MYSQL_SERVER 1
```

Comment:
Why is this needed?

Reply:
Some classes needed by the module are defined only when MYSQL_SERVER is defined. We have this define in a number of places around storage engines. Will add a comment explaining why it's needed here.